Sloth.Web/Controllers/TeamsController.cs

```diff
@@ -1,13 +1,15 @@
 using System;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using Sloth.Core;
 using Sloth.Core.Extensions;
 using Sloth.Core.Models;
 using Sloth.Web.Identity;
 using Sloth.Web.Models.TeamViewModels;
+using Sloth.Web.Resources;
 
 namespace Sloth.Web.Controllers
```

Comment:
I don't know the answer but I wonder what would happen if you added `[Authorize(Policy = PolicyCodes.TeamAdmin)]` at the controller level, and then on just the Index() method you added `[Authorize]`.  Would it revert back to allowing anyone?  Or maybe if you made a new Policy that allows everyone and used it.  I'm assuming based on the way other .net attributes work it would take the one declared on the method to override/supersede the one on the class.

Reply:
Just tried the plain Authorize attribute. Didn't work. Then tried adding a policy with no roles, but the attribute on the action doesn't override the class-level one. It builds upon it. I think the design is the class level one can be more permissible than action-level, but not vice versa.

Reply:
ok, that makes sense and thanks for checking it out